install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +321,89 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
```

Comment:
Here are my nfs bools:
# getsebool -a|grep nfs|grep on\$
nfs_export_all_ro --> on
nfs_export_all_rw --> on
virt_use_nfs --> on

Note: the 2 export bools above can both be set off or mixed on/off with no effect on R/W access to my nfs export.